x-pack/docs/en/security/securing-communications/security-basic-setup.asciidoc

```diff
@@ -4,9 +4,11 @@
 <titleabbrev>Set up basic security</titleabbrev>
```

Comment:
I have a feeling that the sectioning of this manual installation guide is currently not very helpful.

The following section (I mean the whole `security-basic-setup.asciidoc` file) only covers Elasticsearch cluster TLS - the other stack components (Kibana, Beats) are somewhat covered in the following section that talks about HTTPS .
I could maybe be convinced that this is really about the "Stack" if it would also describe setting the passwords for the built-in users eg `kibana` and/or `metricbeat_monitoring`.

Reply:
@albertzaharovits, I don't disagree, and there's a larger effort (tracked in [#75153](https://github.com/elastic/elasticsearch/issues/75153)) for incorporating the security documentation for other Stack components (such as Logstash and Beats) into the Elasticsearch security docs. I consider that work tangential to this PR, but not part of it.